kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```diff
@@ -50,18 +50,22 @@ spec:
                       message: "kind should be unset or 'KafkaProxy'."
                   properties:
                     group:
-                      description: "The group of the referent KafkaProxy"
+                      description: |-
+                        Group is the group of the referent. If specified it must be, "kroxylicious.io".
+                        When unspecified, the "kroxylicious.io" API group is inferred.
                       type: string
                       maxLength: 253
                       pattern: ^$|^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
                     kind:
```

Comment:
The description of the `proxyRef` is "Specified the KafkaProxy this ingress belongs to" 

- (trivial point) typo
- (main point) it's really weird to then have this `kind` property (and `group`). I know we want to keep the door open to supporting other `proxyRef` types in the future, but I think to do that it's sufficient that the array items are `object`. They just need a `name` for now. We can add the extra properties with their back-compat defaults when we come to support other things.
- If we're serious about supporting other types then we should think really hard whether the name for this should be `proxyRef`, since in the future the other type might not be called "proxy", at which point it's a bit confusing. A neutral name like `parentRef` might be a better choice.

Reply:
For this first PR I think it's better to go minimal and remove the fields, we can build it up later. I don't think there's a future for `parentRef` but we could discuss async and rename in a subsequent PR.